core/tests/data/dummy_ci_tests/e2e_actions_dummy.yml

```diff
@@ -8,3 +8,9 @@ jobs:
       - name: Run Additional Player E2E Test
         if: startsWith(github.head_ref, 'update-changelog-for-release') == false
         run: python -m scripts.run_e2e_tests --suite="twoWords" --prod_env
+      - name: Run Additional Editor E2E Test
```

Comment:
Just a quick question -- why are we adding this file (and other dummy tests)?

Reply:
These are needed for backend testing of check_e2e_tests_are_captured_in_ci_test.py file. PTAL at the test I have written for more details.